packages/taboule/public/index.html

```diff
@@ -35,21 +39,23 @@ <h2>Taboule</h2>
           <label>Show Params Inputs</label>
           <input type="checkbox" checked="true" />
         </div>
+        <button id="taboule-reload">Reload Taboule</button>
```

Comment:
I would move the "reload" feature inside `Taboule`, so it can be fired as well when imported as standalone component

Reply:
<img width="1629" alt="image" src="https://user-images.githubusercontent.com/17749195/195151380-ebbb0eac-574f-496b-9bf0-ed18e4474b21.png">